gemini/evaluation/get_started_with_genai_model_eval_service.ipynb

```diff
@@ -0,0 +1,1744 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.            score = -1.0
</pre><p>Is there another way to write this besides a try/except?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/553/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>

Reply:
<p>the try/except is needed. but I refractor the function so now looks more readable. </p>